<ui version="4.0" >
 <class>search_engine</class>
 <widget class="QWidget" name="search_engine" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Search</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupEngines" >
       <property name="minimumSize" >
        <size>
         <width>131</width>
         <height>132</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>125</width>
         <height>132</height>
        </size>
       </property>
       <property name="title" >
        <string>Search Engines</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="mininova" >
          <property name="text" >
           <string/>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="piratebay" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="isohunt" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="meganova" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="search_lbl" >
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="font" >
            <font>
             <family>Sans Serif</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text" >
            <string>Search Pattern:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="search_pattern" >
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="search_button" >
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>29</height>
            </size>
           </property>
           <property name="text" >
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stop_search_button" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>29</height>
            </size>
           </property>
           <property name="text" >
            <string>Stop</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="status_lbl" >
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="font" >
            <font>
             <family>Sans Serif</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text" >
            <string>Status:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="search_status" >
           <property name="minimumSize" >
            <size>
             <width>400</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="font" >
            <font>
             <family>Sans Serif</family>
             <pointsize>9</pointsize>
             <weight>50</weight>
             <italic>true</italic>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text" >
            <string>Stopped</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="results_lbl" >
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font" >
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text" >
          <string>Results:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>721</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="resultsBrowser" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="contextMenuPolicy" >
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="autoScroll" >
        <bool>true</bool>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="indentation" >
        <number>1</number>
       </property>
       <property name="itemsExpandable" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="download_button" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Download</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_button" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>601</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="update_nova_button" >
         <property name="text" >
          <string>Update search plugin</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>search_pattern</sender>
   <signal>returnPressed()</signal>
   <receiver>search_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>421</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel" >
     <x>685</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
